Gemfile

```diff
@@ -20,6 +20,7 @@ gem "minima", "~> 2.1", ">= 2.1.1"
 # If you have any plugins, put them here!
 group :jekyll_plugins do
    gem "jekyll-feed", "~> 0.9", ">= 0.9.2"
+   gem "jemoji", "~> 0.12.0", ">= 0.12.0"
```

Comment:
Did you even end up using this gem? What does it do differently than copy-pasta an emoji into markdown? They're just unicode characters afterall

Reply:
Yep, I'm using it... IMHO it makes it easier to add in emoji's